Week2/HW2/8.sameType.js

```diff
@@ -0,0 +1,47 @@
+let x = 11;
+console.log (x);
+
+let y = 'It is rainy today.';
+console.log (y);
+
+let z = ['rose', 'tulip', 'daisy', 'orchid'];
+console.log (z);
+
+let t = false;
+console.log (t);
+
+if ( typeof x === typeof y) {
+    console.log (`${x} and  ${y} are in the same type`)
+} else { 
+    console.log (`${x} and  ${y} are not in the same type`)
+};
+
+if ( typeof x === typeof y) {
```

Comment:
Ah, I think you forgot to change `x` and `y`. Because you do have all the `comparisons` but you have `x` and `y` everywhere:)